src/components/Chat/ReplyMessageText.js

```diff
@@ -164,22 +165,22 @@ const styles = StyleSheet.create({
     fontSize: 12,
     fontFamily: fonts.inter[600],
     lineHeight: 14.53,
-    color: isMe ? '#000' : colors.elm,
+    color: isMe ? COLORS.black : colors.elm,
     marginRight: 5.7,
   }),
   time: (isMe) => ({
     fontSize: 10,
     fontFamily: fonts.inter[600],
     lineHeight: 12,
-    color: isMe ? '#000' : colors.elm,
+    color: isMe ? COLORS.black : colors.elm,
     marginLeft: 5,
   }),
   quotedProfileName: {
     flexDirection: 'row',
     alignItems: 'center',
   },
   message: (isMe) => ({
-    color: isMe ? '#000' : colors.elm,
+    color: isMe ? COLORS.black : colors.elm,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Replacing hardcoded color values with dynamic ones from the theme object improves modularity and consistency. This is a good practice as it allows for easier changes in the future if the color scheme needs to be updated.
```diff
-    color: isMe ? '#000' : colors.elm,
+    color: isMe ? COLORS.black : colors.elm,

-    color: isMe ? '#000' : colors.elm,
+    color: isMe ? COLORS.black : colors.elm,

-    color: isMe ? '#000' : colors.elm,
+    color: isMe ? COLORS.black : colors.elm,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->